hw_2_functions.py

```diff
@@ -0,0 +1,284 @@
+    print(f'You selected "{user_item}". Computer selected "{comp_item}"')
+    # Defining the winner
+    define_winner(user_move, computer_move)
+
+
+print('\nTask 1. Rock-scissors-paper game\n')
+# Running the game in the loop
+while True:
+    play_game_rsp()
+    play_again = input("\nPlay again? Press Y to continue or any other key to stop\n")
+    if play_again.lower() != "y":
+        break
+
+
+# 2)Try to imagine a world in which you might have to stay home for (Corona virus) 14 days at any given time.
+# Do you have enough toilet paper(TP) to make it through?
+# Although the number of squares per roll of TP varies significantly, we'll assume each roll has 500 sheets,
+# and the average person uses 57 sheets per day.
+
+# Create a function that will receive a dictionary with two key/values:
+# "people" ⁠— Number of people in the household.
+# "tp" ⁠— Number of rolls.
+# Return a statement telling the user if they need to buy more TP!
+
+def get_paper_forecast(h_hold, forecast_period=14):
+    """Function accepts following params:
+        1) mandatory parameters people and tp - number of people and number of TP rolls}
+        2) optional parameter forecast_period (days) that has default value of 14 days
+        """
+    sheets_per_roll = 500
+    per_person_daily_sheets = 57
+    consumption = h_hold['people'] * per_person_daily_sheets * forecast_period
+    inventory = h_hold['tp'] * sheets_per_roll
+    sheets_left = consumption - inventory
+    if sheets_left:
+        rolls = sheets_left//sheets_per_roll + 1 if (sheets_left%500) != 0 else sheets_left//500
+    print(f"For the next {forecast_period} days You need {consumption} sheets. You have {inventory} sheets.")
+    if inventory < consumption:
+        return print(f"Alarm! Run to the nearest store and buy {rolls} roll(s) of toilet paper before it's too late\n")
+    else:
+        return print(f"Keep calm and keep pooping. You have enough toilet paper for {forecast_period} day(s):)\n")
+
+
+# Testing TP rolls calculation
+print('\nTask 2. Toilet paper - to buy or not to buy\n')
+household_1 = {'people': 2, 'tp': 1}
+household_2 = {'people': 2, 'tp': 1}
+household_3 = {'people': 6, 'tp': 18}
+household_4 = {'people': 6, 'tp': 6}
+get_paper_forecast(household_1)
+get_paper_forecast(household_4, 21)
+
+
+# 3) Make a function that encrypts a given input with these steps:
+# Input: "apple"
+# Step 1: Reverse the input: "elppa"
+# Step 2: Replace all vowels using the following chart:
+# a => 0
+# e => 1
+# i => 2
+# o => 2
+# u => 3
+# # "1lpp0"
+# Example:
+# encrypt("banana") ➞ "0n0n0b"
+# encrypt("karaca") ➞ "0c0r0k"
+# encrypt("burak") ➞ "k0r3b"
+# encrypt("alpaca") ➞ "0c0pl0"
+
+
+def encrypt_text(text):
+    """Replaces every vowel letter in given text with corresponding digit from the dict"""
+    text = text[::-1]
+    vowels_dict = {'a': 0, 'e': 1, 'i': 2, 'o': 3, 'u': 4}
+    for ch in text:
+        if ch.isalpha() and ch.lower() in vowels_dict.keys():
```

Comment:
You don't need to check if ch.isalpha() as the keys in your vowels_dict contain just letters, so "ch.lower() in vowels_dict.keys()" is enough